### PR TITLE
Fit and report: change test file and add dependency

### PR DESCRIPTION
**Pull request by xiw315**

### CODE REVIEW

File: b4model/fit_and_report.R [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    if (startsWith(tolower(m_type), 'classif')){
+        metric <-'Accuracy'
+        model<- train(X, y, method=method, metric=metric)
+        testPred <- predict(model, Xv)
+        test_acc <- postResample(testPred, yv)
+        errors <- c(1 - model$results$Accuracy, 1 -test_acc[1]) 
+    }
+    return(errors)
+}
```
Comment (andrealee011) [inline b4model/fit_and_report.R]: what is this file for? isn't your function already in R/rb4model.R?

**Review by andrealee011 — CHANGES_REQUESTED** [review]

**Comment by andrealee011** [comment]
also your .Rapp.history files should not be committed

File: b4model/fit_and_report.R (hunk 1 above)
Comment (xiw315) [reply b4model/fit_and_report.R]: Resolved this problem.

**Comment by xiw315** [comment]
Resolved mentioned problem

**Review by andrealee011 — APPROVED** [review]

**Pull request merged**